templates/deployment-zabbix-server.yaml

```diff
@@ -63,7 +67,11 @@ spec:
         - name: zabbix-server
           resources:
 {{- toYaml .Values.zabbixserver.resources | nindent 12 }}
+          {{- if .Values.zabbixserver_image.tag }}
```

Comment:
Hi @scoopex!

Can you change

{{- if .Values.zabbixserver_image.tag }}

by

{{- if .Values.zabbixserver.image.tag }}

?

Reply:
Sure, done :-)